### ISSUE TITLE
Support restarting kafka-server service via kafka-manager

### ISSUE THREAD

**Issue by rangrossman** [yahoo/kafka-manager#205]

**Comment by patelh** [comment]
I don't know of any way to force a restart of kafka server externally.  You may want to look at running kafka via mesos and the modifications made by @joestein to control this via kafka manager.

**Comment by bluebob** [comment]
I don't agree that this is a desirable feature. It'd be simple enough to do (I guess) if you're only talking about the server that the KM instance is running on, but what about the rest of the cluster - would you have to have KM installed on every member of the cluster and go to them individually? (Not nice). Other option would be some kind of (extra) control API. If desirable then I think this'd ideal for a KM fork.
Nope, I think this kind of thing is done better by other routes - in my case, I'm controlling KM via systemd, so it's possible/easy to kick off a restart via Ansible etc.
